src/components/Quotes.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react';
+
+const Quotes = () => {
+  const [quotes, setQuotes] = useState('Loading...');
```

Comment:
- [ ] in managing loading and error, there should be as state to handle them differently, the error state should be false initially while loading state should be true initially